src/specfem2D/prepare_timerun_gpu.f90

```diff
@@ -384,88 +384,105 @@ subroutine init_host_to_dev_variable()
            cote_abs(nelemabs),stat=ier)
   if (ier /= 0 ) stop 'error allocating array abs_boundary_ispec etc.'
 
-  ! stacey boundaries
-  if (STACEY_BOUNDARY_CONDITIONS) then
-    ! user output
-    if (myrank == 0) write(IMAIN,*) '    Stacey boundary elements = ',nelemabs
+   if(STACEY_BOUNDARY_CONDITIONS) then
```

Comment:
hi etienne,

please use a consistent indenting with 2-spaces for the next indent level. your indenting here is all messed up and makes it difficult to read the code and to see in which if/do-statement things are done. please make sure your editor uses spaces and not tabs for indenting.

many thanks,
daniel